scripts/performance/web/index.html

```
@@ -0,0 +1 @@
+<html><head><meta name="viewport" content="width=device-width, initial-scale=1" /></head><body><div id="root"></div><script type="module" src="/main.tsx"></script></body></html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)